cmd/minishift/cmd/config/config_test.go

```diff
@@ -111,3 +114,20 @@ func TestWriteConfig(t *testing.T) {
 		b.Reset()
 	}
 }
+
+func decode(r io.Reader) (config.ViperConfig, error) {
+	var data config.ViperConfig
+	err := json.NewDecoder(r).Decode(&data)
+	return data, err
+}
+
+func encode(w io.Writer, m config.ViperConfig) error {
+	b, err := json.MarshalIndent(m, "", "    ")
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write(b)
+
+	return err
+}
```

Comment:
Can we put these functions in cmd/util or pkg/minishift/util as I do not want to see them duplicated again. We already have these in pkg/minishift/config too.